common/src/main/res/drawable/round_corners_green_10.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~ Copyright 2022 Dash Core Group.
+  ~
+  ~ This program is free software: you can redistribute it and/or modify
+  ~ it under the terms of the GNU General Public License as published by
+  ~ the Free Software Foundation, either version 3 of the License, or
+  ~ (at your option) any later version.
+  ~
+  ~ This program is distributed in the hope that it will be useful,
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  ~ GNU General Public License for more details.
+  ~
+  ~ You should have received a copy of the GNU General Public License
+  ~ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+  -->
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle">
+    <corners android:radius="6dp" />
+    <solid android:color="@color/color_system_green_10" />
+</shape>
```

Comment:
This is the background of the "Current APY = 5.9%" which is a 10% alpha system green with rounded corners.

Reply:
It might be better to use the `rounded_background` with a theme that sets the background and corner radius. That way we won't have a bunch of separate files for all kinds of backgrounds. 
The color can just be put into `values` with an appropriate hex code: https://gist.github.com/lopspower/03fb1cc0ac9f32ef38f4